static/docs/get-started/agenda.md

```diff
@@ -27,11 +27,11 @@ If you have data files or data sets and/or you produce other data files, models,
 data sets and you want to:
 
 - capture and save those <abbr>data artifacts</abbr> the same way we capture
-  code,
-- track and switch between different versions of the data easily,
+  code.
+- track and switch between different versions of the data easily.
 - being able to answer the question of how data artifacts (e.g. ML models) were
-  built in the first place,
-- being able to compare them,
+  built in the first place.
+- being able to compare them.
 - bring best practices to your team and get everyone on the same page.
```

Comment:
Here, first of all note that you'll need to solve conflicts with my previous commits [648c12f @ agenda.md](https://github.com/iterative/dvc.org/pull/490/commits/648c12f76d3da35866fb55c18e90c5405a2cb685#diff-46e8e4ce4470f73ba360cfdd6e43713e) and [34df9cd @ agenda.md](https://github.com/iterative/dvc.org/pull/490/commits/34df9cd08bab0d34213470b398a2d931e4069f67#diff-46e8e4ce4470f73ba360cfdd6e43713e) (though you can probably just overwrite with your changes after we decide the rules).

However, I think we should rewrite the entire list so it's not a weird sandwich between "and you want to:" and "Then you are in a good place!" (below). I think the last sentence in the previous paragraph can be: "If you have data files or datasets and you produce other data files, datasets, models, etc., then you may want to:". Then we can just remove "Then you are in a good place!" entirely. Also, please change "being" for "be" in the couple bullets where it appears, as "being" is not grammatically correct in this list.

Reply:
alright